### PR TITLE
[zh]Add 2022-01-07-kubernetes-is-moving-on-from-dockershim.md

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh]Add 2022-01-07-kubernetes-is-moving-on-from-dockershim.md

Ref #31448

### CODE REVIEW

File: content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+If you're part of a project with dependencies on dockershim, or if you're
+interested in helping with the migration effort, please join us! There's always
+room for more contributors, whether to our transition tools or to our
+documentation. To get started, say hello in the
+[#sig-node](https://kubernetes.slack.com/archives/C0BP8PW9G)
+channel on [Kubernetes Slack](https://slack.kubernetes.io/)!
+-->
+如果你是依赖 dockershim 的项目的一部分，或者如果你有兴趣帮助参与迁移工作，请加入我们！
+无论是我们的迁移工具还是我们的文档，总是有更多贡献者的空间。
+要开始，请在 [Kubernetes Slack](https://slack.kubernetes.io/) 上的
```
Comment (tengqm) [inline content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md:154]:
```suggestion
作为起步，请在 [Kubernetes Slack](https://slack.kubernetes.io/) 上的
```

Comment (mengjiao-liu) [reply content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md]: Updated.

File: content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  one, updating existing code samples, tutorials, and tasks, and producing a
+  migration guide for cluster operators.
+- We are reaching out to the rest of the CNCF community to help prepare them for
+  this change.
+-->
+## 我们的下一个步骤 {#our-next-steps}
+
+根据提供的反馈：
+
+- CNCF 和 1.24 版本团队致力于及时交付 1.24 版本的文档。 这包括像本文这样的包含更多信息的博客文章，
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md:140]:
```suggestion
- CNCF 和 1.24 版本团队致力于及时交付 1.24 版本的文档。这包括像本文这样的包含更多信息的博客文章，
```

File: content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+<!--
+## Final thoughts
+
+As a project, we've already seen cluster operators increasingly adopt other
+container runtimes through 2021. 
+We believe there are no major blockers to migration. The steps we're taking to
+improve the migration experience will light the path more clearly for you.
+-->
+## 最后的想法  {#final-thoughts}
```
Comment (howieyuen) [inline content/zh/blog/_posts/2022-01-07-kubernetes-is-moving-on-from-dockershim.md:165]:
```suggestion
## 最终想法  {#final-thoughts}
```

**Review by howieyuen — CHANGES_REQUESTED** [review]
found some nits, plz check

**Comment by mengjiao-liu** [comment]
All feedbacks addressed.  Thanks for your review.@howieyuen

**Pull request merged**